### ISSUE TITLE
[EN][Achievment]We Are Legendary

### ISSUE THREAD

**Issue by DuGalle** [Atlantiss/BugTracker#7224]
http://www.wowhead.com/achievement=4861/we-are-legendary

I have the Sulfuras legendary and the honored rep with my guild but didn't trigger neither the Sulfuras guild ach nor this meta ach

**Comment by eryczek1615** [comment]
I think that at first you must have honored with guild and then Sulfuras

**Comment by aneq** [comment]
yes he has to, but it doesn't change the fact that it's bugged. This should work both ways
